app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyQuestionScreen.kt

```diff
@@ -57,5 +65,7 @@ fun SurveyQuestionScreenPreview() {
         ),
         pageNumber = 1,
         totalNumberOfPage = 5
-    )
+    ) { _, _ ->
+        // Do nothing
+    }
```

Comment:
[Optional] We could leave this as a single line, no biggie 😉 
```suggestion
    ) { _, _ -> }
```